README.md

```diff
@@ -8,14 +8,67 @@ This action allows caching dependencies and build outputs to improve workflow ex
 
 See ["Caching dependencies to speed up workflows"](https://help.github.com/github/automating-your-workflow-with-github-actions/caching-dependencies-to-speed-up-workflows).
 
+## What's New in V2
+
+* Added support for caching multiple paths,
+
+```yaml
+- name: Cache multiple relative paths
+  uses: actions/cache@v2
+  with:
+    path: |
+      node_modules
+      dist
+    key: ${{ runner.os }}-${{ hashFiles('**/lockfiles') }}
+```
+```yaml
+- name: Cache multiple absolute paths
+  uses: actions/cache@v2
+  with:
+    path: |
+      ~/cache
+      /path/to/dependencies
+    key: ${{ runner.os }}-${{ hashFiles('**/lockfiles') }}
+```
+
+[glob patterns](https://github.com/actions/toolkit/tree/master/packages/glob), 
+
+```yaml
+- name: Cache using glob patterns
+  uses: actions/cache@v2
+  with:
+    path: |
+      **/*.ts
+      **/node_modules
+    key: ${{ runner.os }}-${{ hashFiles('**/lockfiles') }}
+```
+
+and single file caches. 
+
+```yaml
+- name: Cache single file
+  uses: actions/cache@v2
+  with:
+    path: cache.tar
+    key: ${{ runner.os }}
+```
+
+* Increased perfomance and improved cache sizes using `zstd` compression
+> Note this feature is off for Windows runner that are using `bsdtar` (e.g., windows-latest hosted runner) due to a bug in ziping large random files with `bsdtar`
+* Allowed caching for all events with a ref
+> See [events that trigger workflow](https://help.github.com/en/actions/reference/events-that-trigger-workflows) for info on which events do not have a `GITHUB_REF`
```

Comment:
```suggestion
* Allowed caching for all events with a ref, see [events that trigger workflow](https://help.github.com/en/actions/reference/events-that-trigger-workflows) for info on which events do not have a `GITHUB_REF`
```

It might be easier to read if we combine this into one bullet point, the `>` note blocks break up the list.